 Contrib/VC17/Jansson/Jansson.vcxproj | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Contrib/VC17/Jansson/Jansson.vcxproj b/Contrib/VC17/Jansson/Jansson.vcxproj
index d8cbee5..61888e7 100644
--- a/Contrib/VC17/Jansson/Jansson.vcxproj
+++ b/Contrib/VC17/Jansson/Jansson.vcxproj
@@ -80,6 +80,7 @@
     <ClCompile>
       <AdditionalIncludeDirectories>./</AdditionalIncludeDirectories>
       <PreprocessorDefinitions>HAVE_STDINT_H</PreprocessorDefinitions>
+      <RuntimeLibrary>MultiThreaded</RuntimeLibrary>
     </ClCompile>
   </ItemDefinitionGroup>
   <ItemDefinitionGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">
@@ -92,6 +92,7 @@
     <ClCompile>
       <AdditionalIncludeDirectories>./</AdditionalIncludeDirectories>
       <PreprocessorDefinitions>HAVE_STDINT_H</PreprocessorDefinitions>
+      <RuntimeLibrary>MultiThreaded</RuntimeLibrary>
     </ClCompile>
   </ItemDefinitionGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
