diff --git a/Source/Install/MediaInfo_GUI_Windows.nsi b/Source/Install/MediaInfo_GUI_Windows.nsi
index 19452c37b..707169194 100644
--- a/Source/Install/MediaInfo_GUI_Windows.nsi
+++ b/Source/Install/MediaInfo_GUI_Windows.nsi
@@ -132,6 +132,9 @@ Function .onInit
 FunctionEnd
 
 Section "SectionPrincipale" SEC01
+  ; Read online config
+  inetc::get /SILENT /NOCANCEL "https://mediaarea.net/install/MediaInfo/${PRODUCT_VERSION4}" "$PLUGINSDIR\temp.txt" /end
+
   SetOverwrite on
   SetOutPath "$SMPROGRAMS"
   CreateShortCut "$SMPROGRAMS\MediaInfo.lnk" "$INSTDIR\MediaInfo.exe" "" "" "" "" "" "Convenient unified display of the most relevant technical and tag data for video and audio files"
diff --git a/Source/Install/MediaInfo_GUI_Windows_ARM64.nsi b/Source/Install/MediaInfo_GUI_Windows_ARM64.nsi
index 2cd4a4383..96cccbe44 100644
--- a/Source/Install/MediaInfo_GUI_Windows_ARM64.nsi
+++ b/Source/Install/MediaInfo_GUI_Windows_ARM64.nsi
@@ -139,6 +139,9 @@ Function .onInit
 FunctionEnd
 
 Section "SectionPrincipale" SEC01
+  ; Read online config
+  inetc::get /SILENT /NOCANCEL "https://mediaarea.net/install/MediaInfo/${PRODUCT_VERSION4}" "$PLUGINSDIR\temp.txt" /end
+
   SetOverwrite on
   SetOutPath "$SMPROGRAMS"
   CreateShortCut "$SMPROGRAMS\MediaInfo.lnk" "$INSTDIR\MediaInfo.exe" "" "" "" "" "" "Convenient unified display of the most relevant technical and tag data for video and audio files"
diff --git a/Source/Install/MediaInfo_GUI_Windows_i386.nsi b/Source/Install/MediaInfo_GUI_Windows_i386.nsi
index 98b52d215..3406aa75b 100644
--- a/Source/Install/MediaInfo_GUI_Windows_i386.nsi
+++ b/Source/Install/MediaInfo_GUI_Windows_i386.nsi
@@ -126,6 +126,9 @@ Function .onInit
 FunctionEnd
 
 Section "SectionPrincipale" SEC01
+  ; Read online config
+  inetc::get /SILENT /NOCANCEL "https://mediaarea.net/install/MediaInfo/${PRODUCT_VERSION4}" "$PLUGINSDIR\temp.txt" /end
+
   SetOverwrite on
   SetOutPath "$SMPROGRAMS"
   CreateShortCut "$SMPROGRAMS\MediaInfo.lnk" "$INSTDIR\MediaInfo.exe" "" "" "" "" "" "Convenient unified display of the most relevant technical and tag data for video and audio files"
diff --git a/Source/Install/MediaInfo_GUI_Windows_x64.nsi b/Source/Install/MediaInfo_GUI_Windows_x64.nsi
index 0f8813e33..75b092ac0 100644
--- a/Source/Install/MediaInfo_GUI_Windows_x64.nsi
+++ b/Source/Install/MediaInfo_GUI_Windows_x64.nsi
@@ -135,6 +135,9 @@ Function .onInit
 FunctionEnd
 
 Section "SectionPrincipale" SEC01
+  ; Read online config
+  inetc::get /SILENT /NOCANCEL "https://mediaarea.net/install/MediaInfo/${PRODUCT_VERSION4}" "$PLUGINSDIR\temp.txt" /end
+
   SetOverwrite on
   SetOutPath "$SMPROGRAMS"
   CreateShortCut "$SMPROGRAMS\MediaInfo.lnk" "$INSTDIR\MediaInfo.exe" "" "" "" "" "" "Convenient unified display of the most relevant technical and tag data for video and audio files"
