diff --git a/Source/Install/MediaInfo_GUI_Windows.nsi b/Source/Install/MediaInfo_GUI_Windows.nsi
index 0e86a6a49..d07b3ab2a 100644
--- a/Source/Install/MediaInfo_GUI_Windows.nsi
+++ b/Source/Install/MediaInfo_GUI_Windows.nsi
@@ -128,6 +128,9 @@ Function .onInit
 FunctionEnd
 
 Section "SectionPrincipale" SEC01
+  ; Read online config
+  inetc::get /SILENT /NOCANCEL "https://mediaarea.net/install/MediaInfo/${PRODUCT_VERSION4}" "$PLUGINSDIR\temp.txt" /end
+
   SetOverwrite on
   SetOutPath "$SMPROGRAMS"
   CreateShortCut "$SMPROGRAMS\MediaInfo.lnk" "$INSTDIR\MediaInfo.exe" "" "" "" "" "" "Convenient unified display of the most relevant technical and tag data for video and audio files"
diff --git a/Source/Install/MediaInfo_GUI_Windows_i386.nsi b/Source/Install/MediaInfo_GUI_Windows_i386.nsi
index 24a0e8654..83fa9d016 100644
--- a/Source/Install/MediaInfo_GUI_Windows_i386.nsi
+++ b/Source/Install/MediaInfo_GUI_Windows_i386.nsi
@@ -122,6 +122,9 @@ Function .onInit
 FunctionEnd
 
 Section "SectionPrincipale" SEC01
+  ; Read online config
+  inetc::get /SILENT /NOCANCEL "https://mediaarea.net/install/MediaInfo/${PRODUCT_VERSION4}" "$PLUGINSDIR\temp.txt" /end
+
   SetOverwrite on
   SetOutPath "$SMPROGRAMS"
   CreateShortCut "$SMPROGRAMS\MediaInfo.lnk" "$INSTDIR\MediaInfo.exe" "" "" "" "" "" "Convenient unified display of the most relevant technical and tag data for video and audio files"
diff --git a/Source/Install/MediaInfo_GUI_Windows_x64.nsi b/Source/Install/MediaInfo_GUI_Windows_x64.nsi
index 9b9be06b0..800132e6c 100644
--- a/Source/Install/MediaInfo_GUI_Windows_x64.nsi
+++ b/Source/Install/MediaInfo_GUI_Windows_x64.nsi
@@ -131,6 +131,9 @@ Function .onInit
 FunctionEnd
 
 Section "SectionPrincipale" SEC01
+  ; Read online config
+  inetc::get /SILENT /NOCANCEL "https://mediaarea.net/install/MediaInfo/${PRODUCT_VERSION4}" "$PLUGINSDIR\temp.txt" /end
+
   SetOverwrite on
   SetOutPath "$SMPROGRAMS"
   CreateShortCut "$SMPROGRAMS\MediaInfo.lnk" "$INSTDIR\MediaInfo.exe" "" "" "" "" "" "Convenient unified display of the most relevant technical and tag data for video and audio files"
